packages/web-vite/DESIGN-TOKENS.md

```
@@ -0,0 +1,445 @@
+--space-3   /* 12px */
+--space-4   /* 16px */
+--space-5   /* 20px */
+--space-6   /* 24px */
+--space-8   /* 32px */
+--space-10  /* 40px */
+--space-12  /* 48px */
+--space-16  /* 64px */
+```
+
+**Example:**
+```css
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)